util/helper/helper_functions.py

```diff
@@ -0,0 +1,28 @@
+"""Utility Function"""
+
+
+def clean_arguments(request_args):
+    """Parse Values from Restful module arguments"""
+    body = {}
+    for key, value in request_args.parse_args().items():
+        if value is not None:
+            body[key] = value
+    return body
+
+
+def clean_product_queries(request):
+    """Parse queries from GET ALL product endpoint"""
+    query = {}
+    valid_queries = (
+        "price__lte",
+        "price__gte",
+        "quantity__gte",
+        "quantity__lte",
+        "date__lte",
+        "date__gte",
+        "title_icontains",
```

Comment:
```suggestion
        "title__icontains",
```
Needs double underscore